websites/docs-smoke-test/src/content/components/cards.mdx

```diff
@@ -0,0 +1,183 @@
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+
+## All options configured
+
+```jsx
+<Cards clickable narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+```
+
+### Output
+
+<Cards clickable narrow smallTitle>
+  <Card>foo</Card>
+  <Card>bar</Card>
+  <Card>next row unless three aside (narrow) prop passed?</Card>
+  <Card>next row in any case</Card>
+</Cards>
+
+# "Card": the actual card
+
+Properties (for authors):
+ * `title` (string): the card title, no markdown allowed.
+ * `href` (string): the link target of the bottom link and, if `clickable` is set, the whole card. `href` has no effect when neither `clickable` or `textLink` are set.
+ * `textLink` (string): the bottom link text. If not set, no bottom link is rendered at all.
+ * `icon` (component): the component to be rendered into the 48x48px space reserved for an icon. If not set, the space is not reserved.
+
+## Flat with Link
+
+```jsx
+<Cards>
+  <Card
+    title="A regular heading"
+    href="/components/images"
+    textLink="Read more about images...">
+  This is visually flat, has a link to the **images (bold)** page and only the link is a clickable target.
+  </Card>
+</Cards>
+```
+
+### Output
+
+<Cards>
+  <Card
+    title="A regular heading"
+    href="/components/images"
+    textLink="Read more about images...">
+  This is visually flat, has a link to the **images (bold)** page and only the link is a clickable target.
+  </Card>
+</Cards>
+
+## Clickable with Link
+
+<Info>Clickable cards must have a href set. If not that is an error condition in development mode</Info>
+
+```jsx
+<Cards clickable>
+  <Card
+    title="A regular heading"
+    href="/components/images"
+    textLink="Read more about images...">
+  This is visually eleveated, has a link to the **images (bold)** page and the complete card is a clickable target.
+  </Card>
+</Cards>
+```
+
+### Output
+
+<Cards clickable>
+  <Card
+    title="A regular heading"
+    href="/components/images"
+    textLink="Read more about images...">
+  This is visually elevated, has a link to the **images (bold)** page and the complete card is a clickable target.
+  </Card>
+</Cards>
+
+## Flat with Icon
+
+This example assumes that the icon used here is already provided by the MDX provider anyways. But since it's MDX alternative or site-specific icons can be provided by adding an `import` statement in the MDX.
+
+<Warning>
+
+As discussed:
+ * this would be the first place where authors have to learn more JSX syntax and not just the HTML-compatible subset.
+ * Also, from an author experience perspective we need to be aware that referencing an icon that does not exist cannot be caught at runtime but will cause a compilation error.
+
+</Warning>
+
+```jsx
+<Cards smallTitle>
+  <Card
+    icon={Icon.ShoppingList}
```

Comment:
The prop should be a React element. Also, since we don't have icons yet, we should keep the prop "disabled".

```suggestion
    // icon={<Icon.ShoppingList />}
```